pkg/resource-managers/topology.go

```diff
@@ -14,16 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package aws
+package resourcemanagers
 
 import (
+	"context"
 	"fmt"
 	"time"
 
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
 	"github.com/aws/aws-sdk-go/aws/awserr"
-	"github.com/aws/aws-sdk-go/service/ec2"
 	"k8s.io/client-go/tools/cache"
-	"k8s.io/cloud-provider-aws/pkg/providers/v1/iface"
+	awssdk "k8s.io/cloud-provider-aws/pkg/aws-sdk"
```

Comment:
same comment on `-` -- could we name this package `services` or something?

Reply:
That's a good suggestion. Given it's the AWS cloud provider, `services` should be a sufficiently clear reference.